sklearn/metrics/regression.py

```diff
@@ -191,7 +191,7 @@ def mean_absolute_error(y_true, y_pred,
 
 def mean_squared_error(y_true, y_pred,
                        sample_weight=None,
-                       multioutput='uniform_average'):
+                       multioutput='uniform_average', squared=False):
```

Comment:
When `sqaured=True` the squared root is taken. That seems a little strange. Would `square_root` make more sense here?

Reply:
thanks a lot @thomasjpfan , I think we made a mistake here. The default squared should be True, which returns MSE.

Reply:
Thank you @qinhanmin2014 @thomasjpfan . I have changed the values as per your comments.